tests/backend/test_anonymous.py

```diff
@@ -40,24 +42,7 @@ def not_decorated():
 
 @pytest.mark.trio
 async def test_anonymous_has_limited_access(anonymous_backend_sock):
```

Comment:
would be usefull to also have a `test_authenticated_has_limited_access`

(we could typically keep all the 3 tests inside the same `tests/backend/test_access.py` file given how related they are)